### PR TITLE
feat: Wallet transfer from time locked address

### PR DESCRIPTION
**Pull request by TheWaWaR**
## Usage example
``` shell
# Transfer from master private key
wallet transfer \
       --from-account ckt1qyqdfjzl8ju2vfwjtl4mttx6me09hayzfldq8m3a0y \
       --from-locked-address ckt1q3w9q60tppt7l3j7r09qcp7lxnp3vcanvgha8pmvsa3jplykxn3238390r7sv7dzgunt0yc0l7ue5usuymudkscqqqqqqqgqyq0y7qxf \
       --to-address ckt1qyqg7zchpds6lv3v0nr36z2msu2x9a5lkhrq7kvyww \
       --capacity 19999.9999 \
       --tx-fee 0.00001
# Password:
# 0xd3f1df20a4b87c01d77b0ab9877c9270c512ce3e9a4443deb0eb685112fc24e5


# Transfer from derived private key (bip32)
wallet transfer \
       --from-account ckt1qyqd5eyygtdmwdr7ge736zw6z0ju6wsw7rssu8fcve \
       --from-locked-address ckt1q3w9q60tppt7l3j7r09qcp7lxnp3vcanvgha8pmvsa3jplykxn32s08dj83qavwc4l4ceyeta2ryhkahxrndxacqqqqqqqgqyqx3vx8k \
       --to-address ckt1qyqvhx2mvdyujpzhpxxk7p5p7jkl0qynqg2qrgj5el \
       --capacity 1999.9998 \
       --tx-fee 0.0001 \
       --derive-change-address ckt1qyq8r0rrv7hlsvhj7u0l90d6zxn87luzxjys95rgrn
# Password:
# 0xcfb9f660acb2c9c50aa29db04a9d2cc235b47ee94b2df18db5b206dbb6774669
```

### CODE REVIEW

**Comment by janx** [comment]
Is this pr ready for reivew & merge?

File: src/subcommands/tx.rs [hunk 1]
```diff
@@ -595,42 +594,10 @@ fn print_cell_info(
...
-            Ok(None)
-        }
-    })
-}
-
 fn get_keystore_signer(key_store: KeyStore, account: H160, password: String) -> SignerFn {
     Box::new(move |lock_args: &HashSet<H160>, message: &H256| {
         if lock_args.contains(&account) {
-            if message == &SIGHASH_TYPE_HASH {
+            if message == &H256::default() {
```
Comment (driftluo) [inline src/subcommands/tx.rs]: I think `H256::zero()` is better

Comment (TheWaWaR) [reply src/subcommands/tx.rs]: Neither method exists.
https://github.com/nervosnetwork/ckb/tree/develop/util/fixed-hash

I think `h256!("0x0")` is the best way.

Comment (driftluo) [reply src/subcommands/tx.rs]: https://docs.rs/numext-fixed-hash/0.1.4/numext_fixed_hash/struct.H256.html#method.zero

Comment (TheWaWaR) [reply src/subcommands/tx.rs]: > https://docs.rs/numext-fixed-hash/0.1.4/numext_fixed_hash/struct.H256.html#method.zero

It's not used in `ckb`. `ckb` has it's own mini implementation.

**Pull request merged**